components/common/LeftNav.tsx

```diff
@@ -24,8 +24,14 @@ interface LeftNavProps {
     signOutAndClear: () => Promise<void>
 }
 
-const LinkItems: Array<LinkItemProps> = [
-    { name: "Home", icon: BsHouse, href: "/" },
+enum Role {
```

Comment:
I'd prefer
```typescript
type Role = 'ADMIN' | 'USER'
```
We could then also revert the `.eslintrc.json` change, because I'm not completely sure about it's implications.

Reply:
I would like to use an enum here. Then I can use it as a constant when checking role e.g. `role == Role.ADMIN`. When it comes to `.eslintrc.json`, I have replaced eslint's `no-shadow` rule with TypeScript specific rule, which seems to be a correct way: https://typescript-eslint.io/rules/no-shadow/ 

Otherwise, linter will give an error always when an enum is used. Fore sure, we can choose not to use enums at all 🙂

Reply:
Fair enough, let's use enum. 🙏